tools/src/test/scala/scala/scalanative/nir/PrivateMethodsManglingSuite.scala

```diff
@@ -0,0 +1,99 @@
+package scala.scalanative.nir
+import org.scalatest.matchers.should.Matchers
+import scala.scalanative.LinkerSpec
+
+class PrivateMethodsManglingSuite extends LinkerSpec with Matchers {
+  "Nested mangling" should "distinguish private methods from different classes" in {
+    val sources = Map(
+      "A.scala" ->
+        """package xyz
+		  |abstract class A{
+		  | protected def encodeLoop(arg1: Int, arg2: String): String
+		  | final def encode(arg1: Int, arg2: String, arg3: Boolean): String = {
+		  | 	def loop(): String = {
+		  |  		if(arg3) "hello"
+		  |    		else encodeLoop(arg1, arg2)
+		  |  	}
+		  | 	loop()	
+		  | }
+		  | private def foo(x: Int): Int = x + 1
+		  | def bar(x: Int): Int = foo(x)
+		  |}
+		  |
+		  |""".stripMargin,
```

Comment:
These multiline strings, are not aligned correctly.
They use tabs instead of spaces.

Reply:
Good point, fixed